pkg/apis/pipeline/v1beta1/pipelinerun_conversion.go

```diff
@@ -208,6 +216,159 @@ func (ptrs *PipelineTaskRunSpec) convertFrom(ctx context.Context, source v1.Pipe
 	ptrs.ComputeResources = source.ComputeResources
 }
 
+// ConvertTo implements apis.Convertible
+func (prs *PipelineRunStatus) ConvertTo(ctx context.Context, sink *v1.PipelineRunStatus) error {
+	// TODO: handle ducktype conversion using knative func once https://github.com/knative/pkg/issues/2655 is fixed
```

Comment:
It looks like we're going to address this in pipelines now, no? can you link to the pipelines issue?